packages/@n8n/db/src/migrations/common/1763048000000-ActivateExecuteWorkflowTriggerWorkflows.ts

```diff
@@ -0,0 +1,166 @@
+import { ERROR_TRIGGER_NODE_TYPE, EXECUTE_WORKFLOW_TRIGGER_NODE_TYPE } from 'n8n-workflow';
+
+import type { MigrationContext, ReversibleMigration } from '../migration-types';
+
+type Node = {
+	type: string;
+	disabled?: boolean;
+	parameters?: Record<string, unknown>;
+};
+
+type Workflow = {
+	id: string;
+	active: boolean;
+	versionId: string;
+	activeVersionId: string | null;
+	nodes: string | Node[];
+};
+
+/**
+ * Activates all workflows that contain an executeWorkflowTrigger node with at least one parameter,
+ * or an errorTrigger node. Also disables any other trigger nodes within those workflows.
+ */
+export class ActivateExecuteWorkflowTriggerWorkflows1763048000000 implements ReversibleMigration {
+	async up({ escape, runQuery, runInBatches, parseJson }: MigrationContext) {
+		const tableName = escape.tableName('workflow_entity');
+		const idColumn = escape.columnName('id');
+		const versionIdColumn = escape.columnName('versionId');
+		const nodesColumn = escape.columnName('nodes');
+		const activeColumn = escape.columnName('active');
+		const activeVersionIdColumn = escape.columnName('activeVersionId');
+
+		const inactiveWorkflows = `SELECT ${idColumn}, ${nodesColumn}, ${versionIdColumn}, ${activeVersionIdColumn} FROM ${tableName} WHERE ${activeColumn} = false`;
```

Comment:
We could limit this to return only workflow that have one of those nodes using a `like` condition:

```sql
AND (nodes like '%n8n-nodes-base.executeWorkflowTrigger%' OR nodes like '%n8n-nodes-base.errorTrigger%')
```

Would reduce the amount of parsing and data processing a lot